src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -80,4 +81,16 @@ public static void writeToFile(Path file, String content) throws IOException {
         Files.write(file, content.getBytes(CHARSET));
     }
 
+    /**
+     * Writes given string to a file by appending it onto the last line of the file.
+     * Will create the file if it does not exist yet.
```

Comment:
would present tense be better?

Reply:
For this, I follow the documentation style of the `writeToFile` method above. So I think `Writes` will be better?

Reply:
Yea I agree with Jin Hao, but perhaps adding a `a` in `Writes a given string` would be better?